src/pages/process.tsx

```diff
@@ -1,33 +1,29 @@
-import { Box, Card, CardBody, CardHeader, Flex } from '@chakra-ui/react'
-import { Election, OrganizationAvatar, OrganizationDescription, OrganizationName } from '@vocdoni/chakra-components'
-import { OrganizationProvider } from '@vocdoni/react-providers'
-import { PublishedElection } from '@vocdoni/sdk'
 import { useLoaderData } from 'react-router-dom'
+import { ArchivedElection, PublishedElection } from '@vocdoni/sdk'
+import { ElectionProvider, OrganizationProvider, useElection } from '@vocdoni/react-providers'
+import Detail from '~components/Process/Detail'
+import { Flex } from '@chakra-ui/react'
 
-const Vote = () => {
-  const election = useLoaderData() as PublishedElection
+const ProcessView = () => {
+  const { election } = useElection()
+  if (!election || !(election instanceof PublishedElection)) return <></>
```

Comment:
As said before, we should handle all types of elections when rendering them (specially in the explorer).